### ISSUE TITLE
exception:Error: Failed to build C++ embedding:

### ISSUE THREAD

**Issue by Blacktaler** [flutter-tizen/flutter-tizen#542]
I've installed tizen sdk and tizen TV emulator.
flutter-tizen doctor has no issues with tizen
trying to run flutter-tizen and it's giving me this error:

target: tizen_cpp_embedding
exception:Error: Failed to build C++ embedding:
Exception in thread "main" java.lang.NoClassDefFoundError: org/tizen/sbilib/for_cli/sbiplugin/SBIPluginManager
        at org.tizen.ncli.subcommands.build.buildnative.BuildNativeCLICommand.call(BuildNativeCLICommand.java:198)
        at org.tizen.ncli.subcommands.build.buildnative.BuildNativeCLICommand.call(BuildNativeCLICommand.java:48)
        at org.tizen.ncli.subcommands.AbstractSubCommand.runCommand(AbstractSubCommand.java:76)
        at org.tizen.ncli.ide.shell.BuildNativeCLI.execute(BuildNativeCLI.java:107)
        at org.tizen.ncli.ide.shell.AbstractCLI.execute(AbstractCLI.java:91)
        at org.tizen.ncli.ide.shell.Main.run(Main.java:190)
        at org.tizen.ncli.ide.shell.Main.main(Main.java:123)
Caused by: java.lang.ClassNotFoundException: org.tizen.sbilib.for_cli.sbiplugin.SBIPluginManager
        at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:419)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:352)
        ... 7 more
#0      throwToolExit (package:flutter_tools/src/base/common.dart:10:3)
#1      NativeEmbedding.build (package:flutter_tizen/build_targets/embedding.dart:124:7)
<asynchronous suspension>
#2      _BuildInstance._invokeInternal (package:flutter_tools/src/build_system/build_system.dart:861:9)
<asynchronous suspension>
#3      Future.wait.<anonymous closure> (dart:async/future.dart:518:21)
<asynchronous suspension>
#4      _BuildInstance.invokeTarget (package:flutter_tools/src/build_system/build_system.dart:799:32)
<asynchronous suspension>
#5      Future.wait.<anonymous closure> (dart:async/future.dart:518:21)
<asynchronous suspension>
#6      _BuildInstance.invokeTarget (package:flutter_tools/src/build_system/build_system.dart:799:32)
<asynchronous suspension>
#7      Future.wait.<anonymous closure> (dart:async/future.dart:518:21)
<asynchronous suspension>
#8      _BuildInstance.invokeTarget (package:flutter_tools/src/build_system/build_system.dart:799:32)
<asynchronous suspension>
#9      FlutterBuildSystem.build (package:flutter_tools/src/build_system/build_system.dart:628:16)
<asynchronous suspension>
#10     TizenBuilder.buildTpk (package:flutter_tizen/tizen_builder.dart:104:11)
<asynchronous suspension>
#11     TizenDevice.startApp (package:flutter_tizen/tizen_device.dart:352:7)
<asynchronous suspension>
#12     FlutterDevice.runHot (package:flutter_tools/src/resident_runner.dart:476:33)
<asynchronous suspension>
#13     HotRunner.run.<anonymous closure> (package:flutter_tools/src/run_hot.dart:432:14)
<asynchronous suspension>
#14     Future.wait.<anonymous closure> (dart:async/future.dart:518:21)
<asynchronous suspension>
#15     HotRunner.run (package:flutter_tools/src/run_hot.dart:463:34)
<asynchronous suspension>
#16     RunCommand.runCommand (package:flutter_tools/src/commands/run.dart:808:27)
<asynchronous suspension>
#17     FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1389:27)
<asynchronous suspension>
#18     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#19     CommandRunner.runCommand (package:args/command_runner.dart:212:13)
<asynchronous suspension>
#20     FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:355:9)
<asynchronous suspension>
#21     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#22     FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:295:5)
<asynchronous suspension>
#23     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:119:9)
<asynchronous suspension>
#24     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#25     main (package:flutter_tizen/executable.dart:84:3)
<asynchronous suspension>

**Comment by swift-kim** [comment]
Looks like an issue with the Tizen Studio. Would you try reinstalling all installed packages or Tizen Studio itself?

**Comment by Blacktaler** [comment]
will uninstalling and re-installing tizen studio solve the problem?
Just I faced a lot of problems during the installation of tizen-studio on windows.
flutter-tizen could not find tizen-sdk, then I created a new path named TIZEN_SDK, then pasted the path of tizen-studio\tools\ide\bin, after that it could not find the location of Gcc 9.2 toolchain and Native app development cli of Wearable 5.5.
Then I pasted only the path of tizen-studio. Finally flutter-tizen doctor didn't show any issue.
